Packages/net.yutopp.vgltf.unity/Runtime/MeshImporter.cs

```diff
@@ -36,10 +36,12 @@ public async Task<IndexedResource<Mesh>> Import(int meshIndex, GameObject go, Ca
             var gltf = Context.Container.Gltf;
             var gltfMesh = gltf.Meshes[meshIndex];
 
-            return await Context.Resources.Meshes.GetOrCallAsync(meshIndex, async () =>
-            {
-                return await ForceImport(meshIndex, go, ct);
-            });
+            var mesh = await Context.Resources.Meshes.GetOrCallAsync(meshIndex,
```

Comment:
I think that it will be better to create a local instance and pass it to these functions explicitly instead of using AuxResources.
The local instance will be `PritmitivesMap` like `Dictionary<PrimitiveResourceKey, PrimitiveResource>`. 

All resources under AuxResources (in general, `Context.Resources`) are managed and will be disposed, thus we should avoid managing same instances under one context.

Reply:
One more, disposing resources managed under `Context.Resources` manually should be also avoided 🙏